public/.eslintcache

```diff
@@ -0,0 +1 @@
+[{"C:\\code\\work\\waterfront\\public\\src\\index.js":"1","C:\\code\\work\\waterfront\\public\\src\\app.js":"2","C:\\code\\work\\waterfront\\public\\src\\reportWebVitals.js":"3"},{"size":526,"mtime":1609789416235,"results":"4","hashOfConfig":"5"},{"size":685,"mtime":1609789416233,"results":"6","hashOfConfig":"5"},{"size":402,"mtime":1609202889928,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"xy8jas",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"15"},"C:\\code\\work\\waterfront\\public\\src\\index.js",[],["16","17"],"C:\\code\\work\\waterfront\\public\\src\\app.js",[],"C:\\code\\work\\waterfront\\public\\src\\reportWebVitals.js",[],["18","19"],{"ruleId":"20","replacedBy":"21"},{"ruleId":"22","replacedBy":"23"},{"ruleId":"20","replacedBy":"24"},{"ruleId":"22","replacedBy":"25"},"no-native-reassign",["26"],"no-negated-in-lhs",["27"],["26"],["27"],"no-global-assign","no-unsafe-negation"]
```

Comment:
just pointing this out for you

Reply:
Thanks I've removed this file and added it to the gitignore